### DISCUSSION TITLE
minimize the left side of twitter

### DISCUSSION THREAD

**Discussion by Sportsmen** [insin/control-panel-for-twitter#354] [General]
I am probably an idiot but I could not figure out how to do this. I would like to get the left side "minimized so I just see the icons. Currently, it looks like this. I appreciate the help!

<img width="4974" alt="image" src="https://github.com/insin/control-panel-for-twitter/assets/24534158/ec1b739a-ffa3-4789-bc65-7719093ab46a">

**Comment by doom99** [comment]
Similar - I typically have my desktop browser (firefox) full screen on one monitor.  While I like the expanded view this extension provides, I'd much prefer if it could pin the twitter nav column to the left rather than letting it float with window size (whether text or just icons as above request).  Not skilled in CSS or uBlock enough to hack a change myself.
